tests/problems.py

```diff
@@ -152,6 +159,15 @@ def analytical_sample(self, y0, ts, bm):
     def nfe(self):
         return self._nfe
 
+class Ex1Additive(Ex1):
```

Comment:
I don't think you can make `Ex1` to have additive noise like this. Additive noise SDEs are defined such that the diffusion should not depend on the state, i.e. the derivative be 0. The original `Ex1` doesn't satisfy this.

Reply:
Sure, done.